examples/text-generation/requirements_awq.txt

```
@@ -0,0 +1,2 @@
+triton==3.1.0
+autoawq
```

```suggestion
autoawq
transformers>=4.48.2,<4.49.0
```
Let's also have the dependency on Transformers in main